pkg/storaged/fsys-tab.jsx

```diff
@@ -501,25 +546,30 @@
         extract_option(split_options, "noauto");
         const opt_ro = extract_option(split_options, "ro");
         const opt_never_auto = extract_option(split_options, "x-cockpit-never-auto");
+        const opt_nofail = extract_option(split_options, "nofail");
+        const opt_netdev = extract_option(split_options, "_netdev");
         const split_options_for_fix_config = split_options.slice();
         if (forced_options)
             for (const opt of forced_options)
                 extract_option(split_options, opt);
 
         let mount_point_text = null;
         if (old_dir) {
-            if (old_opts && old_opts != "defaults") {
-                let opt_texts = [];
-                if (opt_ro)
-                    opt_texts.push(_("read only"));
-                if (opt_never_auto)
-                    opt_texts.push(_("never mounted at boot"));
-                opt_texts = opt_texts.concat(split_options);
-                if (opt_texts.length)
-                    mount_point_text = cockpit.format("$0 ($1)", old_dir, opt_texts.join(", "));
-                else
-                    mount_point_text = old_dir;
-            } else
+            let opt_texts = [];
+            if (opt_ro)
+                opt_texts.push(_("read only"));
+            if (opt_never_auto)
+                opt_texts.push(_("never mount at boot"));
+            else if (opt_netdev)
+                opt_texts.push(_("after network"));
+            else if (opt_nofail)
+                opt_texts.push(_("ignore failure"));
+            else
+                opt_texts.push(_("stop boot on failure"));
+            opt_texts = opt_texts.concat(split_options);
+            if (opt_texts.length)
+                mount_point_text = cockpit.format("$0 ($1)", old_dir, opt_texts.join(", "));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)